### PR TITLE
Acknowledge community contribution in changelog

### PR DESCRIPTION
**Pull request by antkmsft**

### CODE REVIEW

**Comment by antkmsft** [comment]
Thank you @jnyfah!

File: .vscode/cspell.json [hunk 1]
```diff
@@ -89,6 +90,7 @@
     "iusg",
     "jepio",
     "Jeremi",
+    "jnyfah",
```
Comment (LarryOsterman) [inline .vscode/cspell.json:93]: Does it make sense to put these into the changelog directly?  That way we don't have to constantly update the cspell.json file for every contributor.

Something like is done for the readme file in attestation?
```md
<!-- cspell:words opentelemetry -->
```

**Review by LarryOsterman — APPROVED** [review]

File: .vscode/cspell.json (hunk 1 above)
Comment (antkmsft) [reply .vscode/cspell.json:93]: I like the idea of moving cspell exceptions closer to the places where they are applied. For code, that should be a safe way to go.
But in this case, I am not sure if automation that compiles release notes is going to handle the comment correctly.
Let me to complete this PR as it is, and I will open an issue to review and move exceptions to the files.

Comment (antkmsft) [reply .vscode/cspell.json:93]: #4668

**Pull request merged**